src/System.Private.CoreLib/shared/System/Number.BigInteger.cs

```diff
@@ -353,7 +353,7 @@ public static void Add(ref BigInteger lhs, uint value, ref BigInteger result)
                 while (index < lhsLength)
                 {
                     ulong sum = (ulong)(lhs._blocks[index]) + carry;
-                    lhs._blocks[index] = (uint)(sum);
+                    result._blocks[index] = (uint)(sum);
```

Comment:
👍, this isn't failing in production because we only ever call it as `Add(ref this, value, ref this);`